html/partials/compare/repositories.html.haml

```diff
@@ -11,23 +11,10 @@
       .col-xs-11
         %h2
           Repositories
-          .scope-summary
-            - if @diff["repositories"].only_in1
-              %span.summary-part
-                #{@description_a.name}: #{@diff["repositories"].only_in1.length} repositories
-            - if @diff["repositories"].only_in2
-              %span.summary-part
-                #{@description_b.name}: #{@diff["repositories"].only_in2.length} repositories
-            - if @diff["repositories"].changed
-              %span.summary-part
-                %a{ href: "#repositories_changed" }
-                  changed
-                = ": #{@diff["repositories"].changed.length} repositories"
-            - if @diff["repositories"].common
-              %span.summary-part
-                %a.show-common-elements{ href: "#repositories_both" }<
-                  both
-                = ": #{@diff["repositories"].common.length} repositories"
+          = render_partial "compare/summary",
+            scope: "repositories",
+            singular: "repository",
+            plural: "repositories"
```

Comment:
Hm, to me the inflection doesn't really belong to the plugin definition file. 

What I have on my list for some time, though, is to make our pluralize method a bit smarter so that it can handle the `repository` -> `repositories` case. That would also simplify that code a bit.